CONTRIBUTING.md

```diff
@@ -23,3 +23,34 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Building Dagger from the command line
+
+* Go to the [Bazel installation page](https://docs.bazel.build/versions/master/install.html) and install Bazel.
+  * Learn more about Bazel targets [here](https://docs.bazel.build/versions/master/build-ref.html).
+* Clone the Dagger repository: `git clone git@github.com:google/dagger.git dagger`
```

Comment:
Is this implicit?

Reply:
When I follow step I like not having to think about anything. Copy paste, copy paste. This achieves that ;)

Reply:
With GitHub now somehow favoring HTTPS over SSH, this should probably be the HTTPS URL then so it works for everyone. Just sayin'